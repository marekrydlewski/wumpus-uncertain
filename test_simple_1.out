1.00 0.00 0.00 0.30 0.46 0.00 0.46 0.00 
0.00 0.00 0.00 0.00 0.30 0.30 0.00 0.00 
0.42 0.00 0.00 0.00 1.00 0.00 1.00 0.00 
0.00 0.83 0.00 0.00 0.00 0.30 0.00 0.00 
0.00 0.00 0.42 0.00 0.30 0.30 0.42 0.00 
0.00 0.00 0.00 0.00 0.42 0.30 0.00 0.83 
0.00 0.00 0.00 0.83 0.00 0.00 0.00 0.00 
0.00 0.00 0.00 0.00 0.42 0.30 0.00 0.42 
