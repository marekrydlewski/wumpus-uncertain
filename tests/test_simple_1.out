0.20 0.20 0.34 0.20 
0.20 0.34 0.00 0.34 
0.20 0.20 0.34 0.20 
0.20 0.20 0.20 0.20 
